docs/lib/predictions/fragments/android/interpret.md

```diff
@@ -2,49 +2,54 @@ The following API allows you to analyze text for language, entities (places, peo
 
 ## Set up your backend
 
-If you haven't already done so, run `amplify init` inside your project and then `amplify add auth` (we recommend selecting the *default configuration*).
-
 Run `amplify add predictions`, then use the following answers:
 
 ```console
 ? Please select from one of the categories below (Use arrow keys)
-  Identify
-  Convert
-❯ Interpret
-  Infer
-  Learn More
-
+  `Interpret`
 ? What would you like to interpret? (Use arrow keys)
-❯ Interpret Text
-
+  `Interpret Text`
 ? Provide a friendly name for your resource
-  <Enter a friendly name here>
-
+  `interpretText`
 ? What kind of interpretation would you like? (Use arrow keys)
-  Language
-  Entity
-  Keyphrase
-  Sentiment
-  Syntax
-❯ All
-
+  `All`
 ? Who should have access? (Use arrow keys)
-  Auth users only
-❯ Auth and Guest users
+  `Auth and Guest users`
 ```
 
-Run `amplify push` to create the resources in the cloud
+Run `amplify push` to create the resources in the cloud.
 
 ## Working with the API
 
 Here is an example of sending text for interpretation such as sentiment analysis or natural language characteristics.
 
+<amplify-block-switcher>
+<amplify-block name="Java">
+
 ```java
-    public void interpret(String text) {
-        Amplify.Predictions.interpret(
-                text,
-                result -> Log.i("AmplifyQuickstart", result.toString()),
-                error -> Log.e("AmplifyQuickstart", error.toString(), error)
-        );
-    }
+Amplify.Predictions.interpret(
+        "I like to eat spaghetti",
+        result -> Log.i("MyAmplifyApplication", result.getSentiment().getValue().toString()),
+        error -> Log.e("MyAmplifyApplication", error.toString(), error)
+);
+```
+
+</amplify-block>
+<amplify-block name="Kotlin">
+
+```kotlin
+Amplify.Predictions.interpret(
+    "I like to eat spaghetti",
+    { result -> Log.i("MyAmplifyApplication", result.getSentiment()!!.getValue().toString()) },
```

Comment:
Need another annotation on the `getSentiment()` result, to get rid of this platform typing (`!!`)

Reply:
Same as above, I think.